### PR TITLE
Standardise the node runtime used by TS lambdas

### PR DESCRIPTION
**Pull request by rupertbates**
## What does this change?
This PR extracts the node runtime used by Typescript lambdas into a const that can be imported into CDK lib files. this means that we only need to update one place when we want to upgrade the node version our lambdas are running on.

I have tested all the lambdas which have been upgraded to Node 20 as a result of this change on CODE.

### CODE REVIEW

**Review by johnduffell — APPROVED** [review]
good idea 👍

**Review by AndreaDiotallevi — APPROVED** [review]
More consistent - great!

File: cdk/lib/node-version.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { Runtime } from 'aws-cdk-lib/aws-lambda';
+
+// If you change this version make sure you update the version in the .nvmrc file as well
```
Comment (jamesgorrie) [inline cdk/lib/node-version.ts:3]: Nice comment - worth leaving one in the .nvmrc file too?

**Review by jamesgorrie — APPROVED** [review]
Nice and easy

File: cdk/lib/node-version.ts (hunk 1 above)
Comment (rupertbates) [reply cdk/lib/node-version.ts:3]: I tried but the file format doesn't allow comments unfortunately.

**Pull request merged**